tests/containers/multi_runtime.pm

```diff
@@ -16,12 +16,12 @@ use containers::common qw(install_packages);
 use Utils::Logging 'save_and_upload_log';
 
 my $port = 8000;
+my $test_image = "registry.opensuse.org/opensuse/nginx";
```

Comment:
I would use the vanilla tumbleweed image (which has `curl` preinstalled) and any reason why this needs to be on the module scope and not in the subroutine?

Reply:
> I would use the vanilla tumbleweed image (which has `curl` preinstalled) and any reason why this needs to be on the module scope and not in the subroutine?

We use the nginx image to test connectivity.  Here we're reusing it to avoid downloading too much stuff.  It's a global variable because we call `run_tests`.